api/db/changelog/db.changelog-101-billing-project-garbage-collection.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="changelog-101-billing-project-garbage-collection">
+
+    <createTable tableName="billing_project_garbage_collection">
+      <column name="fireCloudProjectName" type="varchar(80)">
```

Comment:
snake_case instead of camelCase please

Reply:
Good catch, thanks.  Oddly enough, this still works.